decidim-elections/app/views/decidim/votings/votings/show.html.erb

```diff
@@ -1,20 +1,22 @@
+<% add_decidim_meta_tags(title: translated_attribute(current_participatory_space.title)) %>
+
 <%
 edit_link(
-  resource_locator(current_voting).edit,
+  resource_locator(current_participatory_space).edit,
   :update,
   :voting,
-  voting: current_voting
+  voting: current_participatory_space
 )
 %>
 
-<div class="wrapper">
-  <div class="row column">
-    <% Decidim::ContentBlock.published.for_scope(:voting_landing_page, organization: current_organization).where(scoped_resource_id: current_voting.id).each do |content_block| %>
-      <% next unless content_block.manifest %>
+<% landing_content_blocks.each do |content_block| %>
+  <% next unless content_block.manifest %>
```

Comment:
Could this check be moved to the controller, where we filter the `landing_content_blocks`?

Reply:
Yes it makes sense, I'll move it to the controller 😁